syntax/javascript.vim

```diff
@@ -136,7 +136,7 @@ syntax cluster javascriptDocs                  contains=javascriptDocParamType,j
 
 if main_syntax == "javascript"
   syntax sync clear
-  syntax sync ccomment javascriptComment minlines=200
+  syntax sync ccomment javascriptComment minlines=200 linebreaks=2
```

Comment:
There are some cases where highlighting wasn't being updated when editing multi-line expressions. For example:
```
let foo = {
  *
  bar(█
```
would only highlight as a method after exiting insert mode (or modifying previous line). With this, it will scan two lines back to look for a match. I am not sure what the performance impact of this is, to be honest; is it slows things down it could be removed, just a minor inconvenience.